tests/foreman/ui/test_hostgroup.py

```diff
@@ -57,12 +57,14 @@ def test_update_hostgroup(self, new_name):
         with Session(self.browser) as session:
             make_hostgroup(session, name=name)
             self.assertIsNotNone(self.hostgroup.search(name))
-            self.hostgroup.update(name, new_name=new_name)
-            self.assertIsNotNone(self.hostgroup.search(new_name))
+            for new_name in generate_strings_list(len1=4):
+                with self.subTest(new_name):
+                    self.hostgroup.update(name, new_name=new_name)
+                    self.assertIsNotNone(self.hostgroup.search(new_name))
+                    name = new_name  # for next iteration
 
-    @data(*generate_strings_list(len1=256))
```

Comment:
Yes, I understand that running all data points may not take so long to run.  But more valid question here is: Does it make sense to run this seven times?  The requirement here is that the name field should error out if the number of characters in the input is more than 255.  It doesn't matter if the input is utf8 or alphanumeric or html or anything.  Satellite considers all input as string and anything more than 255 characters in length will be errored out.  I am not sure what kind of bugs can be caught if we run this 7 times vs. 1 time?  Do you have an idea about that please?